classes/Parameters/UpgradeConfiguration.php

```diff
@@ -95,23 +106,23 @@ public function getChannelZip(): ?string
 
     public function getLocalChannelXml(): ?string
     {
-        return $this->get('archive_xml');
+        return $this->get(self::ARCHIVE_XML);
     }
 
     /**
      * Get the version included in the new release.
      */
     public function getLocalChannelVersion(): ?string
     {
-        return $this->get('archive_version_num');
+        return $this->get(self::ARCHIVE_VERSION_NUM);
     }
 
     /**
      * Get channel selected on config panel (Minor, major ...).
      */
```

Comment:
You can rely on PHPStan to enforce checks on the value as well.
It will require the Upgrader in the `use` list.

```suggestion
    /**
     * Get channel selected on config panel
     * @return Upgrader::CHANNEL_*|null
     */
```